tests/test_ecrv_live.py

```diff
@@ -66,23 +82,37 @@ def test_migrate_live(
     )
 
     strategy_ecrv2 = dev.deploy(StrategyCurveEcrv, vault_ecrv_live)
-    vault_ecrv_live.migrateStrategy(strategy_ecrv_live, strategy_ecrv2, {"from": devychad})
+
+    vault_ecrv_live.updateStrategyDebtRatio(strategy_ecrv_live, 0, {"from": gov_live})
+    strategy_ecrv_live.harvest({"from": dev})
+    vault_ecrv_live.migrateStrategy(strategy_ecrv_live, strategy_ecrv2, {"from": gov_live})
+
+    gauge = "0x3C0FFFF15EA30C35d7A85B85c0782D6c94e1d238"
+    voter_proxy.approveStrategy(gauge, strategy_ecrv2, {"from": gov_live})
+
     genericStateOfStrat(strategy_ecrv_live, token_ecrv, vault_ecrv_live)
     genericStateOfStrat(strategy_ecrv2, token_ecrv, vault_ecrv_live)
     genericStateOfVault(vault_ecrv_live, token_ecrv)
```

Comment:
I would suggest adding balance checks before and after migration, seems like genericState prints the info,
but the values should be asserted against to be sure.